src/main/java/com/example/demo/HelloworldController.java

```diff
@@ -3,11 +3,15 @@
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.List;
+
 @RestController
 public class HelloworldController {
 
-    @GetMapping("/hello")
+    @GetMapping("/helloname")
     public String hello(){
-        return "Hello World";
+        List<String> names=List.of("yagami","ishida","takenouti","izumi","tatikawa","kido","takaishi");
+
+        return "Hello "+names;
```

Comment:
この場合は、こんな感じで、Listをそのまま返す方返してあげるほうが綺麗になると思いました！
```suggestion
    @GetMapping("/names")
    public List<String> names(){
        List<String> names=List.of("yagami","ishida","takenouti","izumi","tatikawa","kido","takaishi");
        return names;
    }
```

Reply:
コメント、ありがとうございます
修正いたしました。
48ddd79f815b40b3d2367432fb95b9a3e0ee0e34